src/controllers/baseHandler.ts

```diff
@@ -48,6 +50,8 @@ export async function baseHandler(
 ): Promise<JSONResponse> {
   const command = lowerCaseMessageCommands(message);
 
+  console.log("Message: ", JSON.stringify(message.data));
+  console.log("Envior:", env);
```

Comment:
?

Reply:
This was added by mistake 
I have fixed it in my new commit

Reply:
ok